core/templates/domain/creator_dashboard/creator-dashboard-backend-api.service.ts

```diff
@@ -129,7 +130,7 @@ export class CreatorDashboardBackendApiService {
     return suggestionThreads;
   }
 
-  _fetchDashboardData(): Promise<CreatorDashboardData> {
+  async _fetchDashboardDataAsync(): Promise<CreatorDashboardData> {
```

Comment:
You'll have to change the name at all callsites also.

Reply:
yes, I'll do that.